nym-vpn-x/src-tauri/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "nym-vpn-x"
+name = "nymvpn-x"
```

Comment:
So I see a lot of renaming nym-vpn-x -> nymvpn-x in the PR, not just here. If this is only done due to the Windows installer, then the correct solution would be to tweak the `installer.nsi` file which sets all these things

Reply:
Are we sure about this, when I tried locally using the old crate name I saw it produced appimage and deb package with the old name nymvpn-x

Reply:
hmm weirdo, I experienced this on both linux and windows (that's why motivated me to rename rust package name)

Reply:
Just tried in dev mode with `nym-vpn-x`
![image](https://github.com/user-attachments/assets/fdf9b55a-75e2-47bc-9f30-4165f61034d0)

Reply:
![image](https://github.com/user-attachments/assets/6d5c101f-1f2a-4836-8da0-61ec6e3b47be)

Reply:
where does the hsts-storage file come from?

Reply:
Double-checked, running the generated appimage, it uses `.local/state/nym-vpn-x` as app data dir

Reply:
![image](https://github.com/user-attachments/assets/06ba458f-4ccb-4691-97cf-baee3eae213f)

Reply:
that's what I get with `nym-vpn-x`

Reply:
yea bundle names respect `productName` but when you run them (the AppImage for instance), still uses `nym-vpn-x` as app dirs (cache config data and so on)